semgrep.yml

```diff
@@ -87,6 +87,24 @@ rules:
       exclude:
         - SPcre.ml
 
+  - id: no-list-map2
```

Comment:
that's the kinda things that are better to put in semgrep.jsonnet to factorize things.
Jsonnet is an experimental feature of semgrep where you can write a rule in jsonnet instead of yaml, which
gives the full power of a full programming language to program your rule.